package.json

```diff
@@ -186,6 +188,7 @@
     "@babel/preset-typescript": "^7.12.13",
     "@testing-library/react-hooks": "^5.1.2",
     "@types/aws-lambda": "^8.10.89",
+    "@types/jsonwebtoken": "^8.5.7",
```

Comment:
I think this should be a [dev dependency](https://medium.com/@dylanavery720/npmmmm-1-dev-dependencies-dependencies-8931c2583b0c)

Reply:
This might also suggest that other `@types` dependencies that are currently in regular dependencies should be moved to dev dependencies

Reply:
yes, but this ticket should just focus on new ones being added

Reply:
@jamescd18 is it a dev dependency if it's under "devDependencies" or is there more to it

Reply:
Nope that's it!